interface.html

```diff
@@ -551,7 +551,9 @@ <h3><small>Apple developer account details and distribution certificates</small>
                   <div class="alert alert-info">
                     <p>If you do not know what the developer account login details are please contact your organization's IT team.</p>
                   </div>
-
+                  <div class="alert alert-warning">
+                    <p>If you have ad blockers enabled for your browser. Please make sure it's disabled.</p>
```

Comment:
I think there's should be a comma (or not a full stop) somewhere in the sentence 💃 

e.g. If you have an Ad blocker enabled on your browser please make sure it's disabled before continuing.

Reply:
Updated

Reply:
@squallstar Needs review again 🙏